lib/ferrum/frame/dom.rb

```diff
@@ -36,6 +36,33 @@ def body
         evaluate("document.documentElement.outerHTML")
       end
 
+      def wait_for_selector(css: nil, xpath: nil, timeout: 3000, interval: 100)
+        evaluate_func(%(
```

Comment:
To be more consistent with the rest of the gem, would you mind changing this to use a `JS` heredoc? This is helpful because many editors support code highlighting in the language specified:

Original:
<img width="472" alt="image" src="https://user-images.githubusercontent.com/7969650/152005111-7fbe8d39-80ee-42d4-ba76-2eadd2b2b1b8.png">


VS:

Subl:
<img width="441" alt="image" src="https://user-images.githubusercontent.com/7969650/152005408-35fc8685-e870-4069-826d-414709df4b0a.png">

Code:
<img width="429" alt="image" src="https://user-images.githubusercontent.com/7969650/152005642-11575244-e065-48cc-bab8-bbd9ade921c3.png">


I haven't tried the code out yet, but I'm looking forward to this feature!

Reply:
it makes sense, will change, thanks!